src/js/components/Menu/index.d.ts

```diff
@@ -31,7 +31,7 @@ export interface MenuProps {
   dropProps?: DropProps;
   gridArea?: GridAreaType;
   icon?: boolean | React.ReactNode;
-  items: ButtonProps[] | ButtonProps[][];
+  items: ButtonProps[] | ButtonProps[][] | ButtonType[] | ButtonType[][];
```

Comment:
I think this can just be
```suggestion
  items: ButtonExtendedProps[] | ButtonExtendedProps[][];
```
like in https://github.com/grommet/grommet/pull/6237/files

Reply:
Yep, I tested it and it did the trick.